theano/tensor/nnet/nnet.py

```diff
@@ -23,6 +23,7 @@
 from theano.tensor import subtensor
 from theano.tensor import elemwise
```

Comment:
The useless block of code you have deleted was the only thing using `elemwise`, that is why flake8 is now complaining.